WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -145,6 +154,10 @@ class LoginEmailViewController: LoginViewController, NUXKeyboardResponder {
         googleLoginButton = button
     }
 
+    func configureGoogleButton() {
+        googleLoginButton?.titleLabel?.font = WPStyleGuide.mediumWeightFont(forStyle: .subheadline)
```

Comment:
I agree, this is not ideal. However, the font needs to be set every time the `traitCollection` changes. Are you suggesting adding one or more methods to `WPStyleGuide+Login` so that code in this class would look similar to this?
`
func configureGoogleButton() {
    WPStyleGuide.styleGoogleButton(googleLoginButton)
}
`